src/components/Cards.js

```diff
@@ -160,18 +120,22 @@ const Content = styled(CardContent)`
 
 function Cards(props) {
 
+  const foldersArray = props.foldersArray;
+  const questions = props.questionsPages;
+  const resources = props.resourcesPages;
+
   if (props.stacked) {
     return (
       <CardWrapper>
-        {cardData.map(card => (
-          <CardItemStacked>
-            <CardLink to={card.to}>
+        {foldersArray.map((card, index) => (
```

Comment:
Maybe break this out into a `Card.jsx` file. Generally it is good practice to not have embedded logic in JS span more than one line. What do you think?

Reply:
Sure, I'll break it out into a separate file.